ui/src/App.tsx

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react"
```

Comment:
This files doesn't need to be change

Reply:
I guess this is because of Prettier :thinking:
changing `'react'` to `"react"` or rather shall I say coverting single quotes to double quotes

Reply:
yeah it's because of Prettier